corgi/tasks/monitoring.py

```diff
@@ -57,8 +57,9 @@ def upsert_interval_task(module, task, hours=None, minutes=None):
     # task at all: https://github.com/celery/django-celery-beat/issues/221
     upsert_cron_task("errata_tool", "load_et_products", hour=0, minute=0)
     upsert_cron_task("prod_defs", "update_products", hour=1, minute=0)
-    upsert_cron_task("pulp", "update_cdn_repo_channels", hour=1, minute=30)
-    upsert_cron_task("rhel_compose", "save_composes", hour=3, minute=0)
+    upsert_cron_task("pulp", "update_cdn_repo_channels", hour=2, minute=0)
+    upsert_cron_task("rhel_compose", "get_builds", hour=3, minute=0)
```

Comment:
I think we should move this to after the save_composes call so that we can process any new builds which were discovered by that task and saved as relations?

Reply:
Thanks, will fix! Agree we should process as many builds as possible - I made my best guess at the dependencies here but wasn't sure what the right order was.